kubeflow/openmpi/master.libsonnet

```diff
@@ -0,0 +1,113 @@
+{
+  all(params):: {
+    kind: "Pod",
```

Comment:
Why is the master just a pod? Isn't that brittle? What happens if the pod fails?

Reply:
You're right. This PR is far from production ready. It only deploys a set of long-running master+workers today, and then the actual mpi command is executed using `kubectl exec`.  Can I address this in a follow-up PR, given that this one is already pretty large?